app/models/user.rb

```diff
@@ -10,6 +10,16 @@ class User < ApplicationRecord
   validates :password, presence: true,
              length: {minimum: Settings.user_valid.min_length_password}
   has_secure_password
+  class <<self
```

Comment:
```suggestion
  class << self
```
thêm dòng trống phía trên